### PR TITLE
refactor(textinput): update component specs

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

`<hx-textinput>` implementing color palette 2.0 styles

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF2158

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by danielmdesigns — CHANGES_REQUESTED** [review]
The text input looks great. My only feedback is that the red "Required" indicator line looks thicker than the default blue active indicator line. Can we adjust that so that they have the same thickness/border weight?

File: src/scss/components/form/mixins/_textControl.scss [hunk 1]
```diff
@@ -14,26 +14,27 @@
...
     font-size: 1rem;
+    font-style: italic;
   }
 }
 
 // Parity: OK
 // Efficiency: OK
 @mixin __textControl--invalid($focused: false) {
-  border: 2px solid $red-900;
+  border-bottom: 2px solid $red-status-500;
```
Comment (100stacks) [inline src/scss/components/form/mixins/_textControl.scss:37]: Please change from `2px` to  `1px` thickness.

File: src/scss/components/form/mixins/_textControl.scss [hunk 2]
```diff
@@ -14,26 +14,27 @@
 // Efficiency: Needs Work
 @mixin __textControl($focused: false) {
   @if $focused {
-    border: 1px solid $cyan-700;
-    box-shadow: $focus-glow;
+    border-bottom: 1px solid $blue-700;
     outline: 0;
+    font-style: normal;
```
Comment (100stacks) [inline src/scss/components/form/mixins/_textControl.scss:19]: Is this style needed?

File: src/scss/components/form/mixins/_textControl.scss [hunk 3]
```diff
@@ -14,26 +14,27 @@
...
 
     background-color: $gray-0;
-    border-radius: 2px;
+    border-radius: 1px;
     border: 1px solid $gray-500;
-    color: $gray-900;
+    color: $gray-800;
     font-family: inherit;
     font-size: 1rem;
+    font-style: italic;
```
Comment (100stacks) [inline src/scss/components/form/mixins/_textControl.scss:30]: Please remove this style.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/form/mixins/_textControl.scss (hunk 1 above)
Comment (lalithkarikelli) [reply src/scss/components/form/mixins/_textControl.scss]: Changes have been done, please review.

File: src/scss/components/form/mixins/_textControl.scss (hunk 2 above)
Comment (lalithkarikelli) [reply src/scss/components/form/mixins/_textControl.scss]: Changes have been done, please review.

File: src/scss/components/form/mixins/_textControl.scss (hunk 3 above)
Comment (lalithkarikelli) [reply src/scss/components/form/mixins/_textControl.scss]: Changes have been done, please review.

**Comment by lalithkarikelli** [comment]
@danielmdesigns  yes, it was 2px , now i have reduced it to 1px . Please review the change . Thanks !

**Review by danielmdesigns — APPROVED** [review]
Text input changes look good. Thanks!

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**